contrib/win32/openssh/win32iocompat.vcxproj

```diff
@@ -361,6 +362,9 @@
     <ClInclude Include="$(OpenSSH-Src-Path)\contrib\win32\win32compat\inc\net\if.h" />
     <ClInclude Include="$(OpenSSH-Src-Path)\contrib\win32\win32compat\inc\time.h" />
     <ClInclude Include="$(OpenSSH-Src-Path)\contrib\win32\win32compat\inc\gssapi.h" />
+    <ClInclude Include="MicrosoftTelemetry.h" />
+    <ClInclude Include="telemetry.h" />
+    <ClInclude Include="telemetryInternal.h" />

```

Comment:
for better readability to distinguish between Microsoft and SSH telemetry files, rename them as
telemetry.h -> sshTelemetry.h
telemetryInternal.h -> sshTelemetryInternal.h
telemetry.c -> sshTelemetry.c